FinHubSA/information-access
# fix: route refresh bug squash

# Pull Request

## Scope
Fixing routing and loading on URL parameters

## Work Done
* dynamic URL routing and loading page directly from URL
* rerouting to home on incorrect searchresults url (only for SR currently!)
* minor style corrections
* moved return button to status bar
* removal of debug statements

## Steps to Test
```npm run start```
Once loaded, refresh the routes for preview, search page and others. Then copy url into new window and run for page. See URL examples below.

## Screenshot
![image](https://user-images.githubusercontent.com/80747408/138275802-9aebc01b-5bc1-486e-95ba-daa883821300.png)
![image](https://user-images.githubusercontent.com/80747408/138275915-58658525-9a66-4fcb-afb3-9d4afce899a9.png)
